common/Geometry2d/Point.hpp

```diff
@@ -258,16 +258,17 @@ class Point {
 
     /**
     * Returns a vector with the same direction as this vector but with magnitude
-    * one.
+    * given,
+    * unless this vector is zero.
     * If the vector is (0,0), Point(0,0) is returned
     */
-    Point normalized() const {
+    Point normalized(float magnitude = 1.0) const {
```

Comment:
I think it's a good change since I think it looks cleaner than seperately multiplying by magnitude and everything works the same with original functionality.